blueocean-dashboard/src/main/resources/jenkins/plugins/blueocean/dashboard/Messages.properties

```diff
@@ -5,6 +5,43 @@ common.date.readable.long=MMM DD YYYY h:mma Z
 common.date.readable.short=MMM DD h:mma Z
 common.pager.loading=Loading...
 common.pager.more=Show more
+# creation
```

Comment:
:ant: I think it would be better to extract that to a new file. It is a pretty isolated component and currently only supported in english. It makes it easier to translate a smaller unit instead the searching in the main translations. 

Then in above components 
`t = i18nTranslator('blueocean-dashboard', 'jenkins.plugins.blueocean.dashboard.IntegrationMessages'); `